README.md

```diff
@@ -10,7 +10,7 @@
 [![Coverage Status](https://codecov.io/gh/pingcap/tidb/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/tidb)
 [![GoDoc](https://img.shields.io/badge/Godoc-reference-blue.svg)](https://godoc.org/github.com/pingcap/tidb)
 
-- [**Slack Channel**](https://pingcap.com/tidbslack/)
+- [**Slack Channel**](https://slack.tidb.io/invite?team=tidb-community&channel=general&ref=github_readme)
```

Comment:
It seems channel #general does not exist, do you mean channel #everyone?

Reply:
PTAL again